patches/chromium/revert_roll_clang_rust_llvmorg-16-init-17653-g39da55e8-3.patch

```diff
@@ -0,0 +1,78 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Charles Kerr <charles@charleskerr.com>
+Date: Wed, 8 Mar 2023 14:10:53 -0600
+Subject: Revert "Roll clang+rust llvmorg-16-init-17653-g39da55e8-3 :
+ llvmorg-17-init-2082-g6d4a674a-1 / [skipping Rust]"
+
+This reverts commit eb840ac593ad7af941acf510ea6ce84c48bde748.
```

Comment:
Is there a path forward for this revert ?

Reply:
@deepak1556  hopefully the next clang roll upstream will resolve this issue.

Reply:
This is the patch that takes us back to clang-16 because clang-17 dies on arm when building SkPatchUtils.cc. Some background discussion on this is in #wg-upgrades on Feb 28 and on Mar 8.

For the path forward, we should report this upstream via the instructions at https://llvm.org/docs/HowToSubmitABug.html#crashing-bugs. They recommend adding a couple of arguments to the command line to coax more debug information out of clang-17, so we may need to do this manually on an arm box.

Reply:
Thanks for the context. I was wondering if there is an upstream bug already or pending :)

Reply:
@deepak1556 I could not find an upstream bug specifically, but https://github.com/llvm/llvm-project/issues/60679 seems like it could be the issue.